toml/src/main/kotlin/org/rust/toml/intentions/TomlElementBaseIntentionAction.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.rust.ide.intentions.RsElementBaseIntentionAction
+import org.rust.toml.tomlPluginIsAbiCompatible
+
+abstract class TomlElementBaseIntentionAction<Ctx> : RsElementBaseIntentionAction<Ctx>() {
```

Comment:
It's a little wierd that `RsElementBaseIntentionAction` is base class for `TomlElementBaseIntentionAction`. Probably, it's worth extracting some language independent implementation to `common` module.
Just a thought, not a call to action